exercises/word-count/word_count_test.py

```diff
@@ -7,81 +7,93 @@ class WordCountTests(unittest.TestCase):
 
     def test_count_one_word(self):
         self.assertEqual(
-            {'word': 1},
-            word_count('word')
+            word_count('word'),
+            {'word': 1}
         )
 
     def test_count_one_of_each(self):
         self.assertEqual(
-            {'one': 1, 'of': 1, 'each': 1},
-            word_count('one of each')
+            word_count('one of each'),
+            {'one': 1, 'of': 1, 'each': 1}
         )
 
     def test_count_multiple_occurences(self):
         self.assertEqual(
-            {'one': 1, 'fish': 4, 'two': 1, 'red': 1, 'blue': 1},
-            word_count('one fish two fish red fish blue fish')
+            word_count('one fish two fish red fish blue fish'),
+            {'one': 1, 'fish': 4, 'two': 1, 'red': 1, 'blue': 1}
+        )
+
+    def test_cramped_list(self):
+        self.assertEqual(
+            word_count('one,two,three'),
+            {'one': 1, 'two': 1, 'three': 1}
+        )
+
+    def test_expanded_list(self):
+        self.assertEqual(
+            word_count('one,\ntwo,\nthree'),
+            {'one': 1, 'two': 1, 'three': 1}
         )
 
     def test_ignores_punctuation(self):
         self.assertEqual(
-            {'car': 1, 'carpet': 1, 'as': 1, 'java': 1, 'javascript': 1},
-            word_count('car : carpet as java : javascript!!&@$%^&')
+            word_count('car : carpet as java : javascript!!&@$%^&'),
+            {'car': 1, 'carpet': 1, 'as': 1, 'java': 1, 'javascript': 1}
         )
 
     def test_include_numbers(self):
         self.assertEqual(
-            {'testing': 2, '1': 1, '2': 1},
-            word_count('testing 1 2 testing')
+            word_count('testing 1 2 testing'),
+            {'testing': 2, '1': 1, '2': 1}
         )
 
     def test_mixed_case(self):
         self.assertEqual(
-            [2, 3],
-            sorted(list(word_count('go Go GO Stop stop').values()))
+            sorted(list(word_count('go Go GO Stop stop').values())),
+            [2, 3]
```

Comment:
Would you mind changing the format of this test to match the other ones for consistency?

I had a poke around and it looks like it was done differently so that words could be uppercase in the list (#207), but the tests added to this track since all assume that words are normalised to lowercase. I kind of like the original idea of giving learners freedom, but I think that practical applications would normalise to lowercase if they wanted a case-insensitive count, and I can't think of a clean and robust way to implement case-insensitive testing.